### DISCUSSION TITLE
Exercise question code snippet unable to view clearly

### DISCUSSION THREAD

**Discussion by harishb00** [Lightning-AI/dl-fundamentals#96] [Exercises] [answered]
Hi @rasbt ,

I could see for many of the exercises and for quizzes were code snippets are used, the text is very hard to view. Characters like paranthesis, commas are hidden.

Below is the example of Unit 4.3 quiz.
![image](https://github.com/Lightning-AI/dl-fundamentals/assets/43300971/0ffa4029-c8ab-4cb9-8e39-154525e57dca)

Could you please add it to the list of issues to be solved?

**Comment by rasbt** [comment, marked as answer]
Thanks for the note. I agree that this looks pretty bad. Will work with our web dev to get it fixed!
